es-space-mgmt/main/default/classes/ReservationManagerController.cls

```diff
@@ -74,4 +74,61 @@ public with sharing class reservationManagerController {
             LIMIT 10
         ];
     }
+
+    @InvocableMethod
+    public static List<Id> updateReservationStatus(
+        List<Requests> recordsToUpdate
+    ) {
+        Schema.SObjectType sObjectType;
+        SObject recordToUpdate;
+        String objectName;
+        List<Id> sucessIds = new List<Id>();
+        if (recordsToUpdate.size() > 0) {
+            // Safe navigate to avoid null pointers
+            sObjectType = recordsToUpdate[0]?.recordId?.getSObjectType();
```

Comment:
Invocable method calls need to be bulkified so you need to process all `recordsToUpdate`.
See the `bulkInvoke/invoke` I used in [Automation Components](https://github.com/trailheadapps/automation-components/blob/master/src-collections/main/default/classes/AddOrInsertRecord.cls).

Reply:
@pozil good call but for this use case, it is always going to be a single record! Let me know if you still think we should bulkify it.

Reply:
On a different note,  since we placed line 88 inside an `if` condition that checks for `recordsToUpdate`'s size, is safe navigation operator really needed there?

Reply:
@adityanaag3 good call! I think one part of the chain can go away. While I will still keep the `recordsToUpdate[0].recordId?.getSObjectType();` since `recordId` can be sent as null as it comes from flow. I will fix this in the next PR.